ci/playbooks/e2e-run.yml

```diff
@@ -5,7 +5,7 @@
   tasks:
     - name: Tagged packages
       ansible.builtin.command:
-        chdir: "{{ ansible_user_dir }}/{{ zuul.project.src_dir }}"
+        chdir: "{{ ansible_user_dir }}/src/github.com/openstack-k8s-operators/ci-framework"
```

Comment:
you might consider adding the directory as a var in this playbook at the top to prevent repeating it 3x in the play

vars:
    ci_framework_dir: "{{ ansible_user_dir }}/src/github.com/openstack-k8s-operators/ci-framework"

Reply:
Done, thanks.